### ISSUE TITLE
APT source whitelist request for llvm-toolchain-trusty-6.0

### ISSUE THREAD

**Issue by Kangz** [travis-ci/apt-source-whitelist#378]
With LLVM 6.0 branched there is a new LLVM APT repo. Would it be possible to have it in the whitelist?

```
  {
    "alias": "llvm-toolchain-trusty-6.0",
    "sourceline": "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-6.0 main",
    "key_url": "http://apt.llvm.org/llvm-snapshot.gpg.key"
  },
```

**Comment by onqtam** [comment]
yes please

**Comment by Morwenn** [comment]
Should be fixed now that #373 has been merged :)
